src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.test.tsx

```diff
@@ -51,40 +78,56 @@ describe('<DefaultTranscriptListItem />', () => {
     rulerTicks: createRulerTicks(),
     expandTranscript: false,
     expandDownload: false,
-    expandMoreInfo: false,
-    toggleTranscriptInfo: toggleTranscriptInfo
+    expandMoreInfo: false
   };
 
-  const renderComponent = (props?: Partial<DefaultTranscriptListItemProps>) =>
-    render(<DefaultTranscriptListItem {...defaultProps} {...props} />);
+  let store: ReturnType<typeof mockStore>;
+
+  const wrapInRedux = (props?: Partial<DefaultTranscriptListItemProps>) => {
+    store = mockStore(mockState);
+    return render(
+      <Provider store={store}>
+        <DefaultTranscriptListItem {...defaultProps} {...props} />
+      </Provider>
+    );
+  };
 
   it('displays unspliced transcript', () => {
-    const { queryByTestId } = renderComponent();
+    const { queryByTestId } = wrapInRedux();
```

Comment:
The interface of this function, `wrapInRedux()` with no arguments, looks strange; the name suggests that something will be wrapped in redux, but the function does not accept anything that it's going to wrap.

In a different test file in this PR, `TranscriptsListItemInfo.test.tsx`, a similar function is called `renderComponent`. Maybe use the same name here? Or change the interface of function such that it accepts a component for wrapping (e.g. `wrapInRedux(<DefaultTranscriptListItem {...defaultProps} />)`).